source/_integrations/zwave_js.markdown

```diff
@@ -139,11 +153,61 @@ Value Notification example:
 }
 ```
 
+## Current Limitations
+
+As this integration is still in the early stages there are some important limitations to be aware of.
+
+- Some advanced CommandClasses are not yet fully implemented in Z-Wave JS. No worries as all the most common devices are already working. You can track status [here](https://github.com/zwave-js/node-zwave-js/issues/6). Actually the only one that impacts HA users is Barrier Operator CC (= garage door controllers) but the good news is that this is [almost finished](https://github.com/zwave-js/node-zwave-js/pull/1337)!
+- Configuration of Z-Wave nodes and/or configuration with the Home Assistant UI is currently not yet implemented. You will need to use another tool, such as [zwavejs2mqtt](https://github.com/zwave-js/zwavejs2mqtt), to manage device configuration.
+- Polling is currently not supported in the integration but will be added soon as a service.
+- Support for setting configuration parameters through service calls is currently not supported but may be added in a later release.
+- There currently is no migration path available from any of the other Z-Wave implementations in Home Assistant. Your Z-Wave network is however stored on your stick so migrating will only require you to redo your device and entity naming.
+
+## Migrating from previous Z-Wave implementations
+
+If you are currently running on the [zwave](https://www.home-assistant.io/integrations/zwave/) or [ozw](https://www.home-assistant.io/integrations/ozw/) Z-Wave integration there is **no need to switch over** to Z-Wave JS. The only thing important to know is that all/most development power currently goes to Z-Wave JS and the previous implementations are still provided as-is. They will **NOT be removed** without proper notice but in time there *might* come technical dependencies that render one or both of those integrations unusable. While both integrations will continue to exist and working, we've marked them as deprecated just to make sure that newcomers start with Z-Wave JS instead of legacy.
```

Comment:
```suggestion
If you are currently running on the [zwave](/integrations/zwave/) or [ozw](/integrations/ozw/) Z-Wave integration there is **no need to switch over** to Z-Wave JS. The only thing important to know is that all/most development power currently goes to Z-Wave JS and the previous implementations are still provided as-is. They will **NOT be removed** without proper notice but in time there *might* come technical dependencies that render one or both of those integrations unusable. While both integrations will continue to exist and working, we've marked them as deprecated just to make sure that newcomers start with Z-Wave JS instead of legacy.
```